src/V3EmitCBase.h

```diff
@@ -52,7 +52,7 @@ class EmitCBase VL_NOT_FINAL {
 public:
     static string voidSelfAssign(const AstNodeModule* modp) {
         const string className = prefixNameProtect(modp);
-        return className + "* const __restrict vlSelf VL_ATTR_UNUSED = static_cast<" + className
+        return className + "& __restrict vlSelf VL_ATTR_UNUSED = *static_cast<" + className
```

Comment:
Probably can be 
```suggestion
        return className + "& const __restrict vlSelf VL_ATTR_UNUSED = *static_cast<" + className
```

Reply:
The `const` qualifiers cannot be applied to a reference. The address of a reference can not be changed either. So removing `const` is safe.